README.md

```diff
@@ -30,34 +30,27 @@ Install with:
 npm i nightwatch
 ```
 
-## Up &amp; Running in 2 Minutes:
-
+## Get started in 60 seconds!
 #### 1. Install Nightwatch from NPM
 
 ```sh
-$ npm install nightwatch --save-dev
-```
-
-#### 2. Install Browser Drivers:
+# from your existing project's root dir
+$ npm init nightwatch
 
-##### Geckodriver (Firefox):
-Geckodriver is the WebDriver service used to drive the [Mozilla Firefox Browser](https://www.mozilla.org/en-US/firefox/new/).
-
-```sh
-$ npm install geckodriver --save-dev
+# if you want to initialize a new project
+$ npm init nightwatch path/to/new/project
 ```
 
-##### Chromedriver:
-Chromedriver is the WebDriver service used to drive the [Google Chrome Browser](https://www.google.com/chrome/).
-```sh
-$ npm install chromedriver --save-dev
-```
+#### 2. Answer a few questions about your preferred setup:
 
-or install everything with one line:
+Choose your preferred:
+- Language-test runner combination
+- Local/remote environment
+- List of browsers (press space to select multiple browsers)
+- Folder name
```

Comment:
can we also add `--yes` flag, where user can skip the questions, and can choose default answers

Reply:
The default answers aren't really intuitive.

Reply:
```
? What is your Language - Test Runner setup? 
? Where do you want to run your e2e tests? 
? Where you'll be testing on? 
? Where do you plan to keep your end-to-end tests? 
? What is the base_url of your project? 
```

Reply:
okay